stetho/src/main/java/com/facebook/stetho/inspector/elements/NodeDescriptor.java

```diff
@@ -34,7 +34,11 @@
 
   void setAttributesAsText(E element, String text);
 
-  void getStyles(E element, StyleAccumulator accumulator);
+  void getStyleRules(E element, StyleRuleAccumulator accumulator);
+
+  void getStyles(E element, String ruleName, StyleAccumulator accumulator);
```

Comment:
I feel like we can improve the naming of the methods and/or interfaces to better highlight the relationship between them. I have a few ideas but I'm not convinced of them:

e.g. renaming `getStyles` to `getStyleRuleStyles` is a bit wordy and weird

renaming `getStyles` to `getRuleStyles` seems pretty good, but a dyslexic nightmare next to `getStyleRules`

renaming `getStyles` to `getStylesForRule` avoids the dyslexia but is still a bit unsatisfying

We could rename `StyleRuleAccumulator` to be `StyleRuleNameAccumulator`, then rename `getStyleRules` to `getStyleRuleNames`, and then leave `getStyles` alone. I think I like this one the best.

Thoughts?